src/lib/Behat/PageObject/TrashPage.php

```diff
@@ -45,23 +44,25 @@ public function __construct(UtilityContext $context)
      */
     public function verifyElements(): void
     {
-        $this->adminList->verifyVisibility();
+        $this->trashTable->verifyVisibility();
     }
 
-    public function verifyIfItemInTrash(string $itemType, string $itemName): void
+    public function verifyIfItemInTrash(string $itemType, string $itemName, bool $elementShouldExist): void
     {
-        Assert::assertFalse(
-            $this->isTrashEmpty() ||
-            !$this->adminList->table->isElementInTable($itemName) ||
-            ($this->adminList->table->isElementInTable($itemName) && $this->adminList->table->getTableCellValue('Type', $itemName) !== $itemType),
-            sprintf('Item %s %s is not in trash', $itemType, $itemName)
+        $isElementInTrash = !$this->isTrashEmpty() &&
+            ($this->trashTable->isElementInTable($itemName) && $this->trashTable->getTableCellValue('Type', $itemName) == $itemType);
+        $elementShouldExistString = $elementShouldExist ? 'n\'t' : '';
+
+        Assert::assertTrue(
```

Comment:
This can stay, but in future usages `Assert::assertEquals($elementShouldExist, $isElementInTrash, ...)` is slightly better - we should use dedicated assertions when it's possible.